File: src/main/node/tokenizer.js
"use strict";

const Pacman = require("./pacman.js");
const Ghost = require("./ghost.js");
const Wall = require("./walls.js");
const Pill = require("./pills.js");

module.exports = 

class Tokenizer {
  static getElement(coords, icon) {
    const types = [ new Pacman, 
                    new Ghost, 
                    new Pill, 
                    new Wall
                  ]
    let result;
    types.forEach( (obj) => {
      let element = obj.getElement(coords, icon);
      if (element) { result = element; }
    });
    return result;
  }
}

File: src/main/python/tokenizer.py
from pacman import Pacman
from ghost import Ghost
from pill import Pill
from wall import Wall


def get_element(coords, icon):
    funcs = [Pacman.get_element,
             Ghost.get_element,
             Pill.get_element,
             Wall.get_element]
    for f in funcs:
        element = f(coords, icon)
        if (element):
            return element
    return None
